jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/RawConfiguration.java

```diff
@@ -57,6 +57,12 @@
     List<String> getExcludesList();
   }
 
+  interface CredHelperConfiguration {
+    Optional<String> getHelperName();
```

Comment:
I think `getHelper()` is good enough, if that doesn't cause a conflict with something.

Reply:
Agreed - however, it'll conflict with the `getHelper()` method required by Gradle. I took a similar approach to the one in `PlatformParameters`. I'm not  a Gradle expert but I think it might struggle to properly deal with `Optional<String>` inputs given it's not `Serializable` ( I think)